.github/dependabot.yml

```diff
@@ -6,19 +6,40 @@
 version: 2
 
 updates:
-  # Enable version updates for Docker
+  # Enable version updates for Docker containers in Openverse API
   - package-ecosystem: 'docker'
     # Look for a `Dockerfile` in the `/openverse-api` directory
     directory: '/openverse-api'
     # Check for updates once a week
     schedule:
       interval: 'weekly'
+    labels:
+      - "💻 aspect: code"
+      - "🧰 goal: internal improvement"
+      - "dependencies"
       
-  # Enable version updates for Python
+  # Enable version updates for Python libs in Openverse API
   - package-ecosystem: 'pip'
     # Look for a `Pipfile` in the `/openverse-api` directory
     directory: '/openverse-api'
     # Check for updates once a week
     schedule:
       interval: 'weekly'
+    labels:
+      - "💻 aspect: code"
+      - "🧰 goal: internal improvement"
+      - "dependencies"
+      - "python"
```

Comment:
What's the python label for, just curious?

Reply:
It's just the label Dependabot used to add for Pip updates to distinguish the language (helpful when using in a repo with multiple languages). It's not a label that we've defined.

Reply:
Oh gotcha! I didn't understand why we would want it given the entire repo is Python but if it's not ours to decide then there you go 🙂